Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
```

Comment:
This should be superfluous given https://github.com/glossier/solidus_retail/pull/4/files#diff-134acb9c8b297af74c81f7263868fad6R27